app/assets/stylesheets/styles/components/guides/_guides.css.scss

```diff
@@ -77,12 +126,29 @@
 
     .right {
       position: absolute;
-      top: 1rem;
-      right: 0;
+      right: 2.5rem;
+      top: 3rem;
     }
   }
 
   .actual-requirements {
+    table {
+      border-spacing: 0px;
```

Comment:
Properties should be ordered border-collapse, border-spacing<br>`0px` should be written without units as `0`